.github/workflows/buildDocker.yml

```diff
@@ -0,0 +1,88 @@
+#  based on https://blog.oddbit.com/post/2020-09-25-building-multi-architecture-im/
+---
+# Interlisp workflow to build a Docker Image that supports multiple architectures
+name: 'Build Maiko Docker image'
+
+# Run this workflow on push to master
+# Other branches can be added it needed. 
+on:
+  push:
+    branches:
+      - master
+
+#  Jobs that are run as part of this workflow.
+jobs:
+  # Job to build the docker image
+  # see: https://github.com/docker/build-push-action
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the branch  
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Setup some environment variables
+      - name: Prepare
+        id: prep
+        run: |
+          # Name of the Docker Image. 
+          DOCKER_IMAGE=${{ secrets.DOCKER_USERNAME }}/${GITHUB_REPOSITORY#*/}
```

Comment:
This needs to change.  DOCKER_USERNAME isn't a direct mapping to the dockerhub repository.  Just use interlisp, no need for anything fancy.